docs/docs/index.md

```diff
@@ -1 +1,58 @@
-Add information related to couchbase plugin
+# Overview
+
+Couchbase plugin is developed to virtualize couchbase data source leveraging the following built-in couchbase technologies:
+
+
+- Cross Data Center Replication (XDCR) allows data to be replicated across clusters that are potentially located in different data centers.
+- Cbbackupmgr allows data to be restored on staging host of Couchbase Server.
+
+Ingest Couchbase
+----------------
+
+1. Couchbase cluster/single instance using (XDCR ingestion mechanism) 
+2. Couchbase backup peice using (CBBACKUPMGR ingestion mechanism) - Zero Touch Production.
+
+### <a id="requirements-plugin"></a>Prerequisites
+**Source Requirements:** Couchbase database user with following privileges
+*   XDCR_ADMIN
+*   DATA_MONITOR
+
+**Staging Requirements**: O/S user with following privileges
+1. Regular o/s user.
+2. Execute access on couchbase binaries [ chmod -R 775 /opt/couchbase ]
+3. Empty folder on host to hold delphix toolkit  [ approximate 2GB free space ]
+4. Empty folder on host to mount nfs filesystem. This is just an empty folder with no space requirements and act as base folder for nfs mounts.
+5. sudo privileges for mount, umount. See sample below assuming `delphix_os` is used as delphix user.
```

Comment:
For some reason these are not rendered as an ordered list.

Reply:
Fixed. Yes, an extra space was required.